packages/cli/src/commands/generate/scaffold/__tests__/scaffold.test.js

```diff
@@ -165,6 +165,18 @@ describe('in javascript (default) mode', () => {
     )
   })
```

Comment:
I just duped what was used for form.js and renamed things accordingly. but it fails the tests with `undefined`![console log of error](https://i.imgur.com/tfrNIE2.png)